nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -253,6 +253,12 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The top-level `termonad-with-packages` alias for `termonad` has been removed.
 
+- The NixOS option `nix.settings.max-jobs` has been changed from auto to 1.
+  This default is the same as documented in manpage for nix.conf.
+
+- The build environments do not set the maximum system load for make and ninja (see (treewide: drop -l$NIX_BUILD_CORES)[https://github.com/NixOS/nixpkgs/pull/192447]).
+  Please check Nix configurations `max-jobs` and `cores` to prevent high system loads.
```

Comment:
Nit: "to prevent high system loads" sounds weird to me, in context of the set of defaults after this PR.

Reply:
Thats true. This entry should be part of PR #192447 . Should I create a separate PR?

Reply:
I only really care about the final docs and final set of defaults.

That's one of my problems with all this &ndash; the piece-by-piece let's-try-this approach without "agreeing" on the overall design change.  I have no idea whether `--max-jobs 1` was what all people had in mind when they :+1: the previous change.